mongo/src/main/java/com/redhat/lightblue/mongo/crud/MongoCRUDController.java

```diff
@@ -683,11 +746,26 @@ private void createUpdateEntityInfoIndexes(EntityInfo ei) {
                 LOGGER.info("Dropping {}",index.get("name"));
                 entityCollection.dropIndex(index.get("name").toString());
             }
+            boolean background = true;
```

Comment:
'background' is misleading. What you're really doing here is determining if a new case insensitive index is to be created. Rename the flag to signify that, and add a comment where i tis used in place of background option.